dash.el

```diff
@@ -111,14 +118,18 @@ Note: `it' is not required in each form."
 (defun -each-indexed (list fn)
   "Call (FN index item) for each item in LIST.
 
-In the anaphoric form `--each-indexed', the index is exposed as symbol `it-index'.
+In the anaphoric form `--each-indexed', the index is exposed as
```

Comment:
I think this sentence can be removed from here and move to the documentation of `--each-indexed` along with the description (the same way you did for the other anaphora).